HealthMonitoring.Management.Core/Registers/EndpointRegistry.cs

```diff
@@ -93,7 +90,6 @@ public bool TryUnregisterById(Guid id)
             endpoint.Dispose();
 
             _endpointConfigurationRepository.DeleteEndpoint(endpoint.Identity.Id);
-            _statsRepository.DeleteStatistics(endpoint.Identity.Id);
```

Comment:
Those stats would become orphan, but stats cleaner will delete them eventually as old stats.
The reason for not deleting those stats here is that it may cause deletion of many records like 20160 if stats are being collected every 30s for 7 days, that would slow down API and may cause issues if db is busy.